Dockerfile

```diff
@@ -1,16 +1,24 @@
 # From: https://github.com/rapidsai/cudf/blob/master/Dockerfile
 FROM cudf
 
+RUN apt install -y zlib1g-dev
+
 ARG CUDA_MAJOR=9
 ARG CUDA_MINOR=2
 RUN source activate cudf && conda install -c pytorch faiss-gpu cuda${CUDA_MAJOR}${CUDA_MINOR}
 RUN source activate cudf && conda install -c anaconda cython
 
 ADD ml-prims /cuML/ml-prims
 ADD cuML /cuML/cuML
+WORKDIR /cuML/cuML
+RUN source activate cudf && \
```

Comment:
This is more of a general question than a change request, since we use the name `rapids` for the environment of the containers, wouldn't we want to name it the same in the repository dockers?

Reply:
I should make a note about this in both the cuDF and cuML repos, but this is intended to be a development container, rather than official RAPIDS release container.

The idea behind this is, you can build cuDF with whatever branches or fixes you need, then extend with a custom cuML build on top.

Reply:
makes total sense :)